src/sagemaker/local/image.py

```diff
@@ -49,6 +49,9 @@
 TRAINING_JOB_NAME_ENV_NAME = "TRAINING_JOB_NAME"
 S3_ENDPOINT_URL_ENV_NAME = "S3_ENDPOINT_URL"
 
+# SELinux Enabled
+SELINUX_ENABLED = os.environ.get("SAGEMAKER_LOCAL_SELINUX_ENABLED", "False").lower() in ["1", "true", "yes"]
```

Comment:
I used an environment variable because I could not find a pythonic way to check if the system is running SELinux. The [selinux](https://pypi.org/project/selinux/) package in pypi failed on my workstation when installed inside a conda environment, so I am discarding that option.